tests/snappi/pfc/files/helper.py

```diff
@@ -519,9 +531,16 @@ def __verify_results(rows,
                                   'Total TX dropped packets {} should be 0'.
                                   format(dropped_packets))
 
-        # Check if pause frames are counted in the PFC counters
-        for peer_port, prios in flow_port_config[1].items():
-            for prio in prios:
-                pfc_pause_rx_frames = get_pfc_frame_count(duthost, peer_port, prio)
+    # Check if pause frames are correctly counted in the PFC counters
+    for peer_port, prios in flow_port_config[1].items():
+        for prio in range(len(prios)):
+            pfc_pause_rx_frames = get_pfc_frame_count(duthost, peer_port, prios[prio])
+            total_egress_packets, _ = get_queue_count(duthost, peer_port, prios[prio])
+            if test_flow_pause and class_enable_vec == class_enable_vector.BIT_SET:
```

Comment:
Why test_flow_pause needs to be true here?

Reply:
This is for the previous test gap and has nothing to do with this test gap - pfc frames should be greater than zero if we expect the queue to be paused - that's what test_flow_pause means